maven.markdown

```diff
@@ -3,46 +3,81 @@ All Smooks components are available from the Maven Central Repository.
 
 If your building Smooks from source using Maven, please use:
 
-1. Java 1.5 or 1.6
-2. [Maven v3](https://maven.apache.org/download.cgi) 
+1. Java 8
+2. [Maven 3](https://maven.apache.org/download.cgi) 
 
+## Artefact IDs
```

Comment:
"Artefact" (with an 'e' instead of an 'i') reads a little strange when talking about maven artifacts/artefacts. I see no reason not to spell it the same as maven.